### PR TITLE
feat(cli): Add init and docs command

### PR DESCRIPTION
**Pull request by tchock**

### CODE REVIEW

File: packages/cli/src/components/Inkuirer/InputQuestionProps.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
```
Comment (JohannesMerz) [inline packages/cli/src/components/Inkuirer/InputQuestionProps.tsx:1]: This filename is confusing to me

File: packages/cli/src/containers/InitContainer/createDecorator.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const fileSystem = getFS();
+
+export function createDecorator(fs: typeof fileSystem, executionPath: string) {
+  const bojagiFolder = path.join(executionPath, '.bojagi');
+  if (!fs.existsSync(bojagiFolder)) {
+    fs.mkdirSync(bojagiFolder);
+  }
+
+  fs.copyFileSync(
+    path.join(__dirname, '../../../boilerplateDecorator.jsx'),
```
Comment (JohannesMerz) [inline packages/cli/src/containers/InitContainer/createDecorator.ts:13]: Path.resolve is better here

File: packages/cli/src/containers/InitContainer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  {
+    type: 'input',
+    name: 'srcFolder',
+    message: '💾 In what folder is your code located',
+    default: './src',
+  },
+  {
+    type: 'select',
+    name: 'createDecorator',
+    message: '📸 Should I create a boilerplate global decorator?',
```
Comment (JohannesMerz) [inline packages/cli/src/containers/InitContainer/index.tsx:32]: Maybe add a link to our docs in here that points to the decorator doc

File: packages/cli/src/components/Inkuirer/InputQuestionProps.tsx (hunk 1 above)
Comment (tchock) [reply packages/cli/src/components/Inkuirer/InputQuestionProps.tsx:1]: I think that was because of the refactor functionality

**Pull request merged**